### PR TITLE
feat(ilayer/catalog): add visible/non-visible badge icon

### PR DESCRIPTION
**Pull request by PhilippeLafreniere18**
**Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines:https://github.com/infra-geo-ouverte/igo2/blob/master/.github/CONTRIBUTING.md#git-commit-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
When a search result layer or a catalog item layer is added to the map, there is no badge to indicate the state of the layer

**What is the new behavior?**
An eye/eye-off logic is implemented to allow the user to see whether or not the layer added to the map is visible

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [X] No

**If this PR contains a breaking change, please describe the impact and migration path for existing applications:**

**Other information**:

### CODE REVIEW

File: packages/geo/src/locale/fr.geo.json [hunk 1]
```diff
@@ -6,6 +6,8 @@
         "layer.addToMapOutRange": "Ajouter à la carte. La donnée est non visible à l'échelle active. Veuillez zoomer +/- si vous voulez l'apercevoir à la carte.",
         "layer.removeFromMap": "Retirer de la carte",
         "layer.removeFromMapOutRange": "Retirer de la carte. La donnée est non visible à l'échelle active. Veuillez zoomer +/- si vous voulez l'apercevoir à la carte.",
+        "layer.removeFromMapNotVisible": "Retirer de la carte. La donnée est visible à l'échelle active, mais n'est pas activée. Veuillez basculer vers l'outil carte pour jouer sur sa visibilité.",
```
Comment (pelord) [inline packages/geo/src/locale/fr.geo.json:9]: remplacer jouer pour "rendre la couche visible"?

File: packages/geo/src/locale/fr.geo.json [hunk 2]
```diff
@@ -6,6 +6,8 @@
...
+        "layer.removeFromMapVisible": "Retirer de la carte. La donnée est déjà visible à l'échelle active. Veuillez basculer vers l'outil carte pour jouer sur sa visibilité.",
```
Comment (pelord) [inline packages/geo/src/locale/fr.geo.json:10]: Plus global, j'enleverais ce state...
image

La poubelle joue déja ce role.

File: packages/geo/src/locale/en.geo.json [hunk 3]
```diff
@@ -6,6 +6,8 @@
         "layer.addToMapOutRange": "Add to map. The layer is not visible at the current scale. Please zoom in/out",
         "layer.removeFromMap": "Remove from map",
         "layer.removeFromMapOutRange": "Remove from map. The layer is not visible at the current scale. Please zoom in/out",
+        "layer.removeFromMapNotVisible": "Remove from map. The layer is visible at the current scale but is not active. Toggle the map tool to adjust layer visibility.",
+        "layer.removeFromMapVisible": "Remove from map. The layer is visible at the current scale. Toggle the map tool to adjust layer visibility.",
```
Comment (pelord) [inline packages/geo/src/locale/en.geo.json:10]: Globally, I would remove this state...
image

The trash can already plays this role.

**Review by pelord — CHANGES_REQUESTED** [review]
see comments.

File: packages/geo/src/locale/fr.geo.json (hunk 2 above)
Comment (pelord) [reply packages/geo/src/locale/fr.geo.json:10]: En fait, pas juste le tooltip, vraiment le state au complet.

File: packages/geo/src/locale/fr.geo.json (hunk 1 above)
Comment (PhilippeLafreniere18) [reply packages/geo/src/locale/fr.geo.json:9]: Mais non ! On joue avec la couche ! C'est comme un jouet pour les géomaticiens :P

**Review by pelord — APPROVED** [review]

**Pull request merged**